packages/gatsby-theme/src/components/Card/Card.stories.mdx

```diff
@@ -76,18 +84,18 @@ A Card component.
       }}
     >
       <Link to='/' variant='default'>
-        <Card.Image
+        <Image
           image={{
             src: "https://placeimg.com/1000/480/tech",
             alt: "tech related stuff",
           }}
         />
       </Link>
       <Card.Heading>Let&#39;s code something now</Card.Heading>
-      <Card.Text sx={{ fontSize: "tiny" }}>
+      <Card.RichText sx={{ fontSize: "tiny" }}>
         <Icon icon='calendar' sx={{ pt: 2, mr: "xsmall" }} />
         Julio 2, 2020
-      </Card.Text>
```

Comment:
I noticed that we have one propType wrong on CardRichText.js, here React is recognizing the children as a type `array` because there are two elements: `Icon` and a string.

Maybe we could change this proptype on Card.RichText?

Reply:
@edvicaty thank you! 🙏